manifests/j/Jidifudi/GT4T/8.49.250222/Jidifudi.GT4T.installer.yaml

```
@@ -0,0 +1,30 @@
+# Automatically updated by the winget bot at 2025/Feb/21
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Jidifudi.GT4T
+PackageVersion: 8.49.250222
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://gt4t.net/apps/gt4t_win_32_bit/GT4T_setup.exe
+  InstallerSha256: 2953247B768E21EB5A30ABC141D9DD1C0AD78FB76D426E4F593DFB3BF6F0E18B
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- Architecture: x64
+  InstallerUrl: https://gt4t.net/apps/gt4t_win_64_bit/GT4T_setup.exe
+  InstallerSha256: F94CB443FB58002206ED348646FD66DA76C9C60A793DE7113565E7304CD9764B

```

```suggestion
 InstallerSha256: 1EDE3E81F2E197FA4B37B60ACA796A332F2301EEF1E9A5F0247EA10555362BCC
```

(Automated response - build 981.)